src/main/java/org/folio/dew/batch/acquisitions/edifact/CompositePOLineConverter.java

```diff
@@ -452,4 +474,37 @@ private String getLocationCode(Location location) {
     }
     return "";
   }
+
+  private BigDecimal calculateCostUnitsTotal(Cost cost) {
+    CurrencyUnit currency = Monetary.getCurrency(cost.getCurrency());
+    MonetaryAmount total = Money.of(0, currency);
+
+    // Physical resources price
+    if (cost.getListUnitPrice() != null && cost.getQuantityPhysical() != null) {
+      MonetaryAmount pPrice = Money.of(cost.getListUnitPrice(), currency)
+        .multiply(cost.getQuantityPhysical());
+      total = total.add(pPrice);
+    }
+    // Electronic resources price
+    if (cost.getListUnitPriceElectronic() != null && cost.getQuantityElectronic() != null) {
+      MonetaryAmount ePrice = Money.of(cost.getListUnitPriceElectronic(), currency)
+        .multiply(cost.getQuantityElectronic());
+      total = total.add(ePrice);
+    }
+    // Discount amount
+    if (cost.getDiscount() != null) {
```

Comment:
AAB shouldn't contain any discounts etc, just unit prices

Reply:
Removed discount and additional cost calculation part, Thanks